src/main/java/com/exactpro/th2/conn/MessageSender.java

```diff
@@ -96,6 +96,19 @@ private void handle(String consumerTag, RawMessageBatch messageBatch) {
     }
 
     private void sendMessage(RawMessage protoMsg) throws InterruptedException {
+        if (protoMsg.getParentEventId().getBookName().equals(untrackedMessagesRoot.getBookName())) {
```

Comment:
The parent event id field and book name can be default or empty and we should pass that message.

Reply:
done